IdentityCore/src/oauth2/aad_v1/MSIDAADV1TokenResponse.m

```diff
@@ -26,6 +26,8 @@
 
 @implementation MSIDAADV1TokenResponse
 
+@dynamic idTokenObj;
```

Comment:
Dynamic is generally used when you're providing setters and getters in runtime or subclasses. Is it necessary here?

Reply:
As discussed, added internal header file for idTokenObj.